backend/api/publishing.py

```
@@ -28,14 +29,194 @@
-router = APIRouter(prefix="/api/scenarios", tags=["Publishing"])
+router = APIRouter(prefix="/api/publishing/scenarios", tags=["Publishing"])
 
 # Performance optimization constants
 DB_EXECUTOR = ThreadPoolExecutor(max_workers=4)
 BATCH_SIZE = 100  # For bulk database operations
 
 # --- SCENARIO PUBLISHING ENDPOINTS ---
 
+@router.get("/", response_model=List[ScenarioPublishingResponse])
+async def get_scenarios(
+    db: Session = Depends(get_db),
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)